blog.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <title>Blog</title>
+      <link href="normalize.css" rel="stylesheet">
+      <link href="styles/blog.css" rel="stylesheet">
+    </head>
+      <body class="container">
+        <section class="header">
+          <li><a href="index(port).html">home.</a></li>
+          <li><a href="portfolio.html">portfolio.</a></li>
+          <li><a href="about.html">about me.</a></li>
+          <li><a href="contact.html">contact.</a></li>
+        </section>
+
+        <section class="title">
+          <h1>Blog Posts<h1>
+        </section>
+
+
+        <section class="blogs">
+          <a href="blog_posts.html">
+            <img src="https://i.imgur.com/WVECL14.jpg" alt="blogpost" /></a>
+        </section>    
+
+
+<!--- Footer -------------->
+        <footer>
+          <ul class="footer">
+            <li>© 2018-</li>
+            <li>contact information: shelandebesai@gmail.com <a href="shelandebesai@gmail.com"</li>
```

Comment:
Your anchor tag isn't correct this should read like:  

`<li>contact information: <a href="shelandebesai@gmail.com"> shelandebesai@gmail.com </a></li>`